frontend/src/components/SidePanel.tsx

```diff
@@ -44,7 +44,7 @@ const css = stylesheet({
     position: 'absolute !important' as any,
     right: 0,
     top: 0,
-    zIndex: 2,
+    zIndex: 10,
```

Comment:
Do you mean creating a single source of all `zIndex` values? that seems like a reasonable idea.

Changing the node `zIndex` was necessary because I wanted the nodes to always be on top of edges (it makes the beginnings of the edges look cleaner), and then the summary card needed to be updated.

The arrowhead and the sidepanel changes have been reverted.

Reply:
Decided to put a single, global `zIndex` object within `Css.tsx`